Assets/HoloToolkit/Sharing/Prefabs/ManualIpConfigUtility.prefab

```diff
@@ -0,0 +1,3126 @@
+%YAML 1.1
```

Comment:
The ability to enter an IP address may be useful in other contexts. For example, sometimes an app needs to connect to a server or another HoloLens on the local network at a specific IP address. Often in an enterprise application using the Unity matchmaking service may not be desired.